themes/digital.gov/layouts/communities/single.html

```diff
@@ -24,108 +24,107 @@ <h1>{{ .Title | markdownify }} Community</h1>
           data-title-heading-level="h3"
           data-root-margin="-350px 0px -350px 0px"
         ></aside>
-        {{/* Summary */}}
-        {{- if .Params.summary -}}
-          <div class="content usa-prose">
-            <p>{{- .Params.summary | markdownify -}}</p>
-
-            {{- if .Params.community_list -}}
-              <section class="join-community-form">
-                <h4 class="join-community-form__header">
-                  <svg
-                    class="join-community-form__header-svg usa-icon dg-icon dg-icon--extra-large margin-bottom-05"
-                    aria-hidden="true"
-                    focusable="false"
-                  >
-                    <use
-                      xlink:href="{{ "uswds/img/sprite.svg#groups" | relURL }}"
-                    ></use>
-                  </svg>
-                  Connect with us
-                </h4>
-
-                {{- range $i, $e := .Params.community_list -}}
-                  <div class="join-community-form__body">
-                    <div class="join-community-form__body-eligibility">
-                      {{/* Checks to see if the community is public or govt only and sets the "terms" of the community, with an option to override the terms for each community by setting it in the front matter */}}
-
-                      {{/* PUBLIC */}}
-                      {{- if eq $e.type "public" -}}
-                        {{/* Checking to see if community terms are defined */}}
-                        {{- if $e.terms -}}
-                          <p>{{- $e.terms | markdownify -}}</p>
-                        {{- else -}}
-                          <p>Open to the public.</p>
+        <section>
+          {{- if .Params.summary -}}
+            <div class="content usa-prose">
```

Comment:
### Issue

For some reason on my local, I see `usa-prose` increasing the `font-size` and `font-weight`. Can you confirm if this is happening for you and address?

<img width="1441" alt="Screenshot 2024-08-01 at 11 30 20 AM" src="https://github.com/user-attachments/assets/44922ac2-afd3-4345-9ae8-a5314a2c0f79">